.github/CODEOWNERS

```diff
@@ -1,2 +1 @@
-# This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
-*   @opensearch-project/notifications
\ No newline at end of file
+*   @xluo-aws @gaobinlong @Hailong-am @SuZhou-Joe @zhichao-aws @Yuye-Zhu
```

Comment:
Yuye has changed his Github name, now it's @yuye-aws, and I have to change the name for the backend plugin.